src/test/scala/com/microsoft/hyperspace/index/rules/HyperspaceRuleTestSuite.scala

```diff
@@ -39,7 +40,13 @@ trait HyperspaceRuleTestSuite extends HyperspaceSuite {
     LogicalPlanSignatureProvider.create(signClass).signature(plan) match {
       case Some(s) =>
         val sourcePlanProperties = SparkPlan.Properties(
-          Seq(),
+          Seq(
+            Relation(
+              Seq("dummy"),
+              Hdfs(Properties(Content(Directory("/")))),
+              "schema",
+              "format",
+              Map())),
```

Comment:
Suggestion: https://github.com/microsoft/hyperspace/pull/142#discussion_r488171512

Reply:
This is required - empty root directory throws exception during the test.